tasks/deprecated-image-check.yaml

```diff
@@ -46,29 +46,34 @@ spec:
           echo "Running conftest using $(params.POLICY_DIR) policy, $(params.POLICY_NAMESPACE) namespace"
           /usr/bin/conftest test --no-fail $(workspaces.sanity-ws.path)/repository_data.json \
           --policy $(params.POLICY_DIR) --namespace $(params.POLICY_NAMESPACE) \
-          --output=json | tee $(workspaces.sanity-ws.path)/deprecated_image_check_output.json || echo "Some tests failed"
+          --output=json 2> $(workspaces.sanity-ws.path)/stderr.txt | tee $(workspaces.sanity-ws.path)/deprecated_image_check_output.json
           echo "Done!"
           exit 0
         elif [ "$http_code" == "404" ];
         then
-          echo "Image not found in Pyxis"
+          echo "Registry/image $(params.IMAGE_REGISTRY)/$(params.IMAGE_REPOSITORY) not found in Pyxis" > $(workspaces.sanity-ws.path)/stderr.txt
+          cat $(workspaces.sanity-ws.path)/stderr.txt
         else
-          echo "Unexpected error (HTTP code $http_code) occured during running conftest"
+          echo "Unexpected error (HTTP code $http_code) occured for registry/image $(params.IMAGE_REGISTRY)/$(params.IMAGE_REPOSITORY) during running conftest" > $(workspaces.sanity-ws.path)/stderr.txt
+          cat $(workspaces.sanity-ws.path)/stderr.txt
         fi
 
     # Format the test results and save them as the HACBS_TEST_OUTPUT Tekton result
     - name: test-format-result
       image: quay.io/redhat-appstudio/hacbs-test:latest
       script: |
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
-          '{result: "ERROR", timestamp: $date}')
+        ERR_MSG="$(cat $(workspaces.sanity-ws.path)/stderr.txt)"
```

Comment:
We could add the `echo -n` to this command to prevent newline issues, like we did [in the Java sast check](https://github.com/redhat-appstudio/build-definitions/blob/main/tasks/sast-java-sec-check.yaml#L64).

Reply:
Yes, thanks! Updated.